recipes/facebook/privacy.json

```diff
@@ -1,5 +1,5 @@
 {
   "url":"https://www.facebook.com/full_data_use_policy",
-  "selector":"body",
+  "selector":".pal",
```

Comment:
It looks to me like a better choice here would be '#content'. An id is preferable to a class because an id can only appear once on a page, whereas a class can match multiple elements on a page.

We should probably rename this file to data_use.json or somesuch.